### PR TITLE
Added loadout: US_USMC_1985

### PR DESCRIPTION
**Pull request by AChesheireCat**
Slightly anachronistic - the loadout is actually based on pictures from the 1991 Liberation of Kuwait.

### CODE REVIEW

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 1]
```diff
@@ -0,0 +1,493 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* FIR AWS(AirWeaponSystem)
+//		* USM Uniforms
+//		* HLC M21
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:9]: NIArms

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		LIST_3("CUP_U_B_BDUv2_roll2_dirty_desert"),
+		"CUP_U_B_BDUv2_dirty_desert"
+	};
+	vest[] = {
+		"usm_vest_pasgtdes_lbe_rm_m"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:19]: Mixed CUP and USM uniforms/vests will clip together. Use either of them, but don't mix.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		"CUP_H_PASGTv2_desert"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:71]: Single line

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+	primaryWeapon[] = {
+		"CUP_arifle_M16A2"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:74]: Single line

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_2("CUP_1Rnd_SmokeRed_M203")
+	};
+	linkedItems[] += {"ACE_Yardage450"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:134]: Yardage on the FTL?

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 6]
```diff
@@ -0,0 +1,493 @@
...
+	headgear[] = {"usm_bdu_8point_des_m"};
+	linkedItems[] = {"ItemGPS"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc119"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:158]: Conflicts with Yardage in the FTL loadout

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 7]
```diff
@@ -0,0 +1,493 @@
...
+		LIST_3("CUP_200Rnd_TE4_Red_Tracer_556x45_M249"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"usm_pack_200rnd_556_bandoliers"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:179]: No AR ammo

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 8]
```diff
@@ -0,0 +1,493 @@
...
+	Items[] += {"ACE_EntrenchingTool"};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : r {
+	displayName = "Rifleman (AT)";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_Mk153Mod0"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:187]: MAT weapon

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 9]
```diff
@@ -0,0 +1,493 @@
...
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class mmgac : aar {
+	displayName = "MMG Ammo Carrier";
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:219]: No MMG ammo

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 10]
```diff
@@ -0,0 +1,493 @@
...
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:228]: No MMG ammo

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 11]
```diff
@@ -0,0 +1,493 @@
...
+	linkedItems[] += {"Binocular"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:229]: Already added by AAR role

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 12]
```diff
@@ -0,0 +1,493 @@
...
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_M2_Tripod_Bag"};
+	linkedItems[] += {"Binocular"};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"CUP_launch_M47"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:251]: Use ACE launcher instead

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 13]
```diff
@@ -0,0 +1,493 @@
...
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" };
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:319]: No quotation marks on macros

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 14]
```diff
@@ -0,0 +1,493 @@
...
+		"SmokeShellPurple"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	headgear[] = {"CUP_H_US_patrol_cap_desert"};
+	goggles[] = {"default"};
+	vest[] = {"CFP_Alice_Vest2"};
+	primaryWeapon[] = {"CUP_srifle_M40A3_bipod"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:361]: Only use preset rifles for backpacks and inventories. In all other cases, use scope[], bipod[] and attachment[]

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 15]
```diff
@@ -0,0 +1,493 @@
...
+		"ToolKit",
+		LIST_3("DemoCharge_Remote_Mag"),
+		LIST_2("SLAMDirectionalMine_Wire_Mag"),
+		"SatchelCharge_Remote_Mag",
+		"ACE_DefusalKit",
+		"ACE_EntrenchingTool",
+	};
+};
+
+class engm : car {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:469]: No minedetector?

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 16]
```diff
@@ -0,0 +1,493 @@
...
+		LIST_2("APERSTripMine_Wire_Mag"),
+		LIST_2("ACE_FlareTripMine_Mag"),
+		LIST_2("ClaymoreDirectionalMine_Remote_Mag")
+	};
+};
+
+class UAV : car {
+	displayName = "UAV Operator";
+	traits[] = {"UAVHacker"};
+	backpack[] = {"B_UAV_01_backpack_F"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:491]: remove backpack

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 17]
```diff
@@ -0,0 +1,493 @@
...
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] = {"Binocular"};
+};
+
+class vg : smg {
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:344]: Misplaced role, move below spotter along with the rest of the vehicle roles.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 18]
```diff
@@ -0,0 +1,493 @@
...
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	headgear[] = {"CUP_H_US_patrol_cap_desert"};
+	goggles[] = {"default"};
+	vest[] = {"CFP_Alice_Vest2"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:360]: Unlisted dependency

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 19]
```diff
@@ -0,0 +1,493 @@
...
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] = {"Binocular"};
+};
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	uniform[] = {"SP_0000_Standard_Coverall_Tan"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:346]: Unlisted dependency

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 20]
```diff
@@ -0,0 +1,493 @@
...
+	vest[] = {"CUP_V_B_Eagle_SPC_Crew"};
+	backpack[] = {};
+	headgear[] = {"SP_CVCHelmet_Tan"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:349]: Unlisted dependency

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 2 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:19]: Swapped to USM as the camoflage, despite being brighter in the arsenal, appears to match the in-game landscapes and lighting better.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 8 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:187]: Replaced with the AT4 (adopted in 1982, iirc).

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 12 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:251]: There's an ACE M47 Dragon? I can't seem to find it in the arsenal, can you provide me the location/item name?

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 18 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:360]: Removed and replaced by the USM LBE.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 19 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:346]: Removed.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp (hunk 12 above)
Comment (Freddo3000) [reply addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:248]: `ace_dragon_super` and the `ace_dragon_sight` CLU. ofc these are the Super Dragon, but it is a lot more authentic in the way you use it compared to the old CUP dragon.

File: addons/tmf_loadouts/CfgLoadouts.hpp [hunk 21]
```diff
@@ -15,6 +15,12 @@ class CfgLoadouts {
 		tooltip = "Author: Freddo";
 		#include "loadouts\us_army_1985.hpp"
 	};
+	class us_usmc_1985 {
+		displayName = "US Marine Corp (1985, DBDU 'Chocolate Chip')";
```
Comment (Freddo3000) [inline addons/tmf_loadouts/CfgLoadouts.hpp:19]: Corps

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 22]
```diff
@@ -0,0 +1,492 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* FIR AWS(AirWeaponSystem)
+//		* USM Uniforms
+//		* NIArms M21
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:9]: M14, the M21 is part of the M14 pack.

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 23]
```diff
@@ -0,0 +1,492 @@
...
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {"usm_bdu_d_m"};
+	vest[] = {
+		"usm_vest_pasgtdes_lbe_rm_m"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:16]: Single line

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 24]
```diff
@@ -0,0 +1,492 @@
...
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = { MEDICAL_R ,"ACE_Flashlight_MX991"};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:50]: remove ItemRadio

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 25]
```diff
@@ -0,0 +1,492 @@
...
+	magazines[] = {
+		LIST_7("CUP_5Rnd_762x51_M24"),
+		LIST_2("HandGrenade"),
+		LIST_4("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:359]: Different vest/headgear compared to Sniper?

File: addons/tmf_loadouts/loadouts/US_USMC_1985.hpp [hunk 26]
```diff
@@ -0,0 +1,499 @@
...
+	displayName = "MAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"ace_dragon_super"};
+	linkedItems[] += {"ace_dragon_sight"};
+};
+
+class matac : car {
+	displayName = "MAT Ammo Carrier";
+	secondaryWeapon[] = {"ace_dragon_super"};
+	};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/US_USMC_1985.hpp:262]: Extra closing bracket. Breaks everything.

**Pull request merged**